Fixes some old problems

TheHllm opened the pull request.

Zaczero commented on 10MinuteMail.net/Types/MailContent.cs:
```diff
@@ -0,0 +1,30 @@
+        [JsonProperty(PropertyName = "subject")]
+        public string Subject { get; set; }
+        [JsonProperty(PropertyName = "datetime")]
+        public string Datetime { get; set; }
```
Naming is a little inconsistent. Some properties are named `DateTime` and some `Datetime`. It would be nice if you could rename all of them to `DateTime` style.

```suggestion
        public string DateTime { get; set; }
```

Zaczero commented on 10MinuteMail.net/Types/MailContent.cs:
```diff
@@ -0,0 +1,30 @@
+        [JsonProperty(PropertyName = "datetime")]
+        public string Datetime { get; set; }
+        [JsonProperty(PropertyName = "datetime2")]
+        public string Datetime2 { get; set; }
```
```suggestion
        public string DateTime2 { get; set; }
```

Zaczero commented on 10MinuteMail.net/Types/MailEntry.cs:
```diff
@@ -0,0 +1,26 @@
+        [JsonProperty(PropertyName = "subject")]
+        public string Subject { get; set; }
+        [JsonProperty(PropertyName = "datetime")]
+        public string Datetime { get; set; }
```
```suggestion
        public string DateTime { get; set; }
```

Zaczero commented on 10MinuteMail.net/Types/MailEntry.cs:
```diff
@@ -0,0 +1,26 @@
+        [JsonProperty(PropertyName = "datetime2")]
+        public string DateTime2 { get; set; }
+        [JsonProperty(PropertyName = "timeago")]
+        public int Timeago { get; set; }
```
```suggestion
        public int TimeAgo { get; set; }
```

Zaczero commented on 10MinuteMail.net/Types/MailResponse.cs:
```diff
@@ -0,0 +1,36 @@
+        [JsonProperty(PropertyName = "mail_get_time")]
+        public int GetTime;
+        [JsonProperty(PropertyName = "mail_get_duetime")]
+        public int Duetime { get; set; }
```
```suggestion
        public int DueTime { get; set; }
```

TheHllm commented:
> [0516643](https://github.com/Zaczero/10MinuteMail.net/commit/051664368edb973259fff37544b805606c08211c)
> This commit seems unnecessary. I can agree with you being added to the `Authors` but the rest should be left as is. I'll update the version number whenever it's a good time to post an update to NuGet =)

Just had that thing in there since i also had put it on nuget.
Removed that commit and added your stuff.
I was not aware you could batch the changes until i was on the "last" commit.
